src/Utilities/ErrorHandling/Assert.hpp

```diff
@@ -33,29 +34,31 @@
 // 20160415) can't figure out that the else branch and everything
 // after it is unreachable, causing warnings (and possibly suboptimal
 // code generation).
-#define ASSERT(a, m)                                                          \
-  do {                                                                        \
-    if (!(a)) {                                                               \
-      disable_floating_point_exceptions();                                    \
-      std::ostringstream avoid_name_collisions_ASSERT;                        \
-      /* clang-tidy: macro arg in parentheses */                              \
-      avoid_name_collisions_ASSERT << m; /* NOLINT */                         \
-      abort_with_error_message(#a, __FILE__, __LINE__,                        \
-                               static_cast<const char*>(__PRETTY_FUNCTION__), \
-                               avoid_name_collisions_ASSERT.str());           \
-    }                                                                         \
+#define ASSERT(a, m)                                                           \
+  do {                                                                         \
+    if (!(a)) {                                                                \
+      disable_floating_point_exceptions();                                     \
+      std::ostringstream avoid_name_collisions_ASSERT;                         \
+      /* clang-tidy: macro arg in parentheses */                               \
+      avoid_name_collisions_ASSERT << std::setprecision(18) << std::scientific \
```

Comment:
Completely optional, since it's unrelated to everything else in the PR:  At some point we switched ERROR to use MakeString.  Changing ASSERT to match would be good.

Reply:
I'll make this change. I'll do it in a new commit but this PR. That way CI needs fewer complete rebuilds :D

Reply:
Okay, never mind... I get annoying non-trivial compiler errors about `operator<<` not existing in `StdHelpers.hpp`. I think we've had this sort of thing before... I'll just leave it for now. This might even be why `ASSERT` is changed but not `ERROR`